doc/api/zlib.md

```diff
@@ -125,16 +168,24 @@ http.createServer((request, response) => {
   // See https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.3
   if (/\bdeflate\b/.test(acceptEncoding)) {
     response.writeHead(200, { 'Content-Encoding': 'deflate' });
-    raw.pipe(zlib.createDeflate()).pipe(response);
+    pipeline(raw, zlib.createDeflate(), response, (err) => {
+      // Handle error.
```

Comment:
@mcollina ... given that `writeHead()` has already been called and at least some of the body may have already been written, we cannot reasonably send a `500` at this point. The best we can do is terminate abruptly and log the error. Is there something else you would recommend here for the example?

Reply:
Explicitly states that we do nothing for the reason you mentioned.